mappings/net/minecraft/util/math/intprovider/IntProvider.mapping

```diff
@@ -0,0 +1,13 @@
+CLASS net/minecraft/class_6017 net/minecraft/util/math/intprovider/IntProvider
+	FIELD field_29945 INT_CODEC Lcom/mojang/serialization/Codec;
+	FIELD field_29946 VALUE_CODEC Lcom/mojang/serialization/Codec;
+	METHOD method_35004 createValidatedCodec (II)Lcom/mojang/serialization/Codec;
```

Comment:
```suggestion
	METHOD method_35004 createValidatingCodec (II)Lcom/mojang/serialization/Codec;
```
Or just `createCodec`. The validation happens when en/decoding, not when the codec is created, hence "validating"